orchagent/pfcactionhandler.h

```diff
@@ -0,0 +1,84 @@
+#ifndef PFC_QUEUE_H
```

Comment:
minor: also change to PFC_ACTION_HANDLER_H or something similar?

Reply:
Done